Justfile

```diff
@@ -15,5 +15,8 @@ format:
 lint: format
 	{{ run_prefix }}ruff check --fix {{ code_dirs }}
 
+typecheck:
+	{{ run_prefix }}mypy src
+
 test:
-	{{ run_prefix }}pytest -x --ff {{ tests_dir }}
\ No newline at end of file
+	{{ run_prefix }}pytest -x --ff {{ tests_dir }}
```

Comment:
Also, but optional, you can move these pytest options into the pyproject.toml done like [here](https://github.com/Tinche/aiofiles/pull/213/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R71-R75) in case you PR goes first.

Reply:
And add the RUF set of rules - https://github.com/Tinche/aiofiles/pull/213/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R141 - just in case and if you like.